command/format/state_test.go

```diff
@@ -101,6 +109,17 @@ func testProviderSchema() *terraform.ProviderSchema {
 					"foo":     {Type: cty.String, Optional: true},
 					"woozles": {Type: cty.String, Optional: true},
 				},
+				BlockTypes: map[string]*configschema.NestedBlock{
+					"nested_attr": {
```

Comment:
Since "attributes" is the _other_ field in `*configschema.Block`, this name may prove confusing for future maintainers; maybe we could just call it `nested`?